includes/Settings/SettingsGeneral.php

```diff
@@ -454,6 +543,57 @@ public function find_templates() {
 		return apply_filters( 'wpo_wcpdf_templates', $installed_templates );
 	}
 
+	public function display_admin_notice_for_shop_address(): void {
+		// Return if the notice has been dismissed.
+		if ( get_option( 'wpo_wcpdf_dismiss_shop_address_notice', false ) ) {
+			return;
+		}
+
+		// Handle dismissal action.
+		if ( isset( $_GET['wpo_dismiss_shop_address_notice'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+			if ( isset( $_GET['_wpnonce'] ) && wp_verify_nonce( sanitize_text_field( wp_unslash( $_GET['_wpnonce'] ) ), 'dismiss_shop_address_notice' ) ) {
+				update_option( 'wpo_wcpdf_dismiss_shop_address_notice', true );
+				wp_redirect( remove_query_arg( array( 'wpo_dismiss_shop_address_notice', '_wpnonce' ) ) );
+				exit;
+			} else {
+				wcpdf_log_error( 'You do not have sufficient permissions to perform this action: wpo_dismiss_requirements_notice' );
+				return;
+			}
+		}
+
+		if (
+			! empty( WPO_WCPDF()->settings->general_settings['shop_address_additional'] ) &&
+			(
+				empty( WPO_WCPDF()->settings->general_settings['shop_address_line_1'] ) ||
```

Comment:
This notice is never displayed:

![image](https://github.com/user-attachments/assets/852e203f-8f41-4c1a-894a-b55029b612ab)